bindings/python/iota_sdk/common.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from json import dumps, JSONEncoder
+from enum import Enum
+import humps
+
+
+def custom_encoder(func, *args, **kwargs):
+    """The routine of dump json string.
```

Comment:
I don't understand this sentence :smile:

Reply:
You already put the dot behind it in the past :trollface:
Is `Converts the parameters to a JSON string.` better?

Reply:
yeah, I guess that would have helped understanding the method better :joy: It also recursively removes `None`s, right? I don't know if that should be mentioned somehow?

Reply:
Added it to the comment